### ISSUE TITLE
Not taking Input In the Level 1 Introduction to Git Commit 

### ISSUE THREAD

**Issue by mohdfarhannawaz** [pcottle/learnGitBranching#1067]
The page of the first exercise Introduction to Git commit is not responding properly as it is not taking the input and not displaying it. Even though I have pressed the key many times. (in the bottom left corner, next to the ' $' sign)
I have refreshed the page many times but still its no good. 
![Screenshot 2023-04-14 164006](https://user-images.githubusercontent.com/101490489/232029479-b95f15d7-004f-4966-81b0-04c95f7fb99b.png)

**Comment by mdel2021** [comment]
Hey, I have the same issue with Chrome 112.0.5615.138. Command Line Bar looks as inactive

**Comment by steady137** [comment]
Experiencing the same on FF 112.0.2

**Comment by pcottle** [comment]
Hrm it must be browser specific, maybe something changed with Firefox. Would love someone to attempt a PR! Otherwise I probably won't have time to look into this

**Comment by NeroBlackstone** [comment]
Opera Version:98.0.4759.39 work, Chromium version:112.0.5615.165

**Comment by PetiPandaRou** [comment]
I just had the same issue on Firefox Nightly 115.0a1 (2023-05-21) (64 bits).   
A refresh of the page resolved the problem.

**Comment by Robloche** [comment]
Same issue here with Chrome 114.0.5735.199 (Build officiel) (64 bits).
Refreshing the page solved the issue.

**Comment by pcottle** [comment]
Yeah I think sometimes just a hard refresh is necessary to kick the page into working...

**Comment by kitswas** [comment]
Still requires multiple refreshes to start taking input.  
Firefox 116.0.3 (64-bit) (Official, Windows 11)
